routes/api.js

```diff
@@ -0,0 +1,21 @@
+/*
+*
+*
+*       Complete the API routing below
+*
+*
+*/
+
+'use strict';
+
+const Translator = require('../components/translator.js');
+
+module.exports = function (app) {
+  
+  let translator = new Translator();
```

Comment:
I notice this line here, but in the PR you created for the example project, you did not actually incorporate this line as you never instantiated `Translator`.  I suppose I am fine leaving it, but I would prefer using `const`.
```suggestion
  const translator = new Translator();
```